llvm/test/CodeGen/SPIRV/pointers/global-addrspacecast.ll

```diff
@@ -1,17 +1,19 @@
-; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
-; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv64-unknown-unknown %s -o - -filetype=obj | spirv-val %}
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv64-unknown-vulkan %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv64-unknown-vulkan %s -o - -filetype=obj | spirv-val %}
```

Comment:
Looks like this mistake is on me. Could you change this to:
`-mtriple=spirv-unknown-vulkan` instead? (Physical + vulkan is weird)